azurelinuxagent/ga/signature_validation_util.py

```diff
@@ -113,7 +115,7 @@ def _write_signature_to_file(sig_string, output_file):
         f.write(binary_signature)
 
 
-def _report_validation_event(op, level, message, name, version, duration):
+def report_validation_event(op, level, message, name, version, duration):
```

Comment:
"- if level is ERROR: prefix message with "[ERROR]" in telemetry" below is no longer accurate